CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- Loading of dogescript files in REPL [#111](https://github.com/dogescript/dogescript/pull/111)
```

Comment:
Love this, what do you think about using tags and https://github.com/dogescript/dogescript/releases/new for this kind of documentation?

Reply:
I totes forgot that was a thing. I think if I tag things correctly with a `milestone` and `enhancement` or `bug` we can then auto-generate the content for that. 

Lemme look at some samples cause i've seen some projects with good notes with 🐛  📦  🚀  style icons for each item type.